include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -518,4 +520,196 @@ def TT_MakeTensorPtrOp : TT_Op<"make_tensor_ptr",
   ];
 }
 
+//
```

Comment:
I assume this may be copy-pasted from the func.func op definition? If so, could you add a link in the comment?

Reply:
Sure, just added